service/controller/chartconfig/v7/key/key.go

```diff
@@ -37,6 +47,18 @@ func HasForceUpgradeAnnotation(customObject v1alpha1.ChartConfig) (bool, error)
 	return result, nil
 }
 
+func IsCordoned(customObject v1alpha1.ChartConfig) bool {
+	_, reasonOk := customObject.Annotations[annotation.CordonReason]
+	_, untilOk := customObject.Annotations[annotation.CordonUntilDate]
+
+	if reasonOk && untilOk {
```

Comment:
That doesn't look right to me. We should check if _now_ is not after _CordonUtilDate_.

Reply:
That would be the part of alarming from cluster-operator, which would check the annotation date. Does it sound okay for you?

Reply:
The presence of the annotations means the resource is cordoned.

The date isn't checked but once the cordon-until date has expired there will be an alert with the reason. So we don't forget about the cordoned CRs and the necessary action can be taken.

@tomahawk28 The metric should actually be exposed by the collector in chart-operator.